docs/source/tutorial/tutorial_template.md

```diff
@@ -73,7 +73,7 @@ The dependencies above might be sufficient for some projects, but for this tutor
 Add the following lines to your `src/requirements.txt` file:
 
 ```text
-kedro[pandas.CSVDataSet, pandas.ExcelDataSet, pandas.ParquetDataSet]==0.18.4   # Specify optional Kedro dependencies
+kedro-datasets[pandas.CSVDataSet, pandas.ExcelDataSet, pandas.ParquetDataSet]~=1.0.0  # Specify optional Kedro dependencies
```

Comment:
Would this be "Specify optional datasets dependencies"?

Reply:
Yes, changed the wording in [d974c6b](https://github.com/kedro-org/kedro/pull/2103/commits/d974c6bee1f2700ef002f874c77287758bc6425e) 🙂.